packages/@aws-cdk/aws-cloudwatch/lib/metric.ts

```diff
@@ -10,8 +10,13 @@ import { normalizeStatistic, parseStatistic } from './private/statistic';
 // eslint-disable-next-line no-duplicate-imports, import/order
 import { Construct } from '@aws-cdk/core';
 
+/**
+ * @deprecated Use `DimensionsMap` instead.
+ */
 export type DimensionHash = {[dim: string]: any};
 
+export type DimensionsMap = {[dim: string]: string};
```

Comment:
I personally prefer the slightly wider formatting of these types, makes it more readable IMO:

```suggestion
export type DimensionsMap = { [dim: string]: string };
```